flink-cdc-e2e-tests/flink-cdc-pipeline-e2e-tests/src/test/java/org/apache/flink/cdc/pipeline/tests/MySqlToElasticsearchE2eITCase.java

```diff
@@ -0,0 +1,379 @@
+            new UniqueDatabase(MYSQL, "mysql_inventory", MYSQL_TEST_USER, MYSQL_TEST_PASSWORD);
+
+    @Parameterized.Parameters(name = "flinkVersion: {0}, elasticsearchVersion: {1}")
+    public static Collection<Object[]> getTestParameters() {
+        List<Object[]> parameters = new ArrayList<>();
+
+        for (String flinkVersion : Arrays.asList("1.19.1", "1.20.0")) {
+            parameters.add(new Object[] {flinkVersion, "6.8.20"});
+            parameters.add(new Object[] {flinkVersion, "7.10.2"});
+            parameters.add(new Object[] {flinkVersion, "8.12.1"});
+        }
+
+        return parameters;
+    }
+
+    @Before
+    public void before() throws Exception {
+        super.before();
+        mysqlInventoryDatabase.createAndInitialize();
+        elasticsearchContainer =
+                new ElasticsearchContainer(elasticsearchVersion)
+                        .withNetwork(NETWORK)
+                        .withNetworkAliases("elasticsearch")
+                        .withLogConsumer(new Slf4jLogConsumer(LOG));
+
+        Startables.deepStart(Stream.of(MYSQL)).join();
+        Startables.deepStart(Stream.of(elasticsearchContainer)).join();
+    }
+
+    @After
+    public void after() {
+        super.after();
+        mysqlInventoryDatabase.dropDatabase();
+        try {
+            dropElasticsearchIndex(mysqlInventoryDatabase.getDatabaseName() + "." + "products");
+            dropElasticsearchIndex(mysqlInventoryDatabase.getDatabaseName() + "." + "customers");
+            if (elasticsearchContainer != null) {
+                elasticsearchContainer.stop();
+            }
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Test
+    public void testSyncWholeDatabase() throws Exception {
+        String databaseName = mysqlInventoryDatabase.getDatabaseName();
+        String pipelineJob =
+                String.format(
+                        "source:\n"
+                                + "  type: mysql\n"
+                                + "  hostname: mysql\n"
+                                + "  port: 3306\n"
+                                + "  username: %s\n"
+                                + "  password: %s\n"
+                                + "  tables: %s.\\.*\n"
+                                + "  server-id: 5400-5404\n"
+                                + "  server-time-zone: UTC\n"
+                                + "\n"
+                                + "sink:\n"
+                                + "  type: elasticsearch\n"
+                                + "  hosts: http://elasticsearch:9200\n"
+                                + "  version: %s\n"
+                                + "  batch.size.max.bytes: 5242880\n"
+                                + "  record.size.max.bytes: 5242880\n"
+                                + "\n"
+                                + "pipeline:\n"
+                                + "  parallelism: 1",
```

Comment:
Could we test in multiple parallelism via `specifiedParallelism`?

Reply:
I’ve updated it now, please have a look. Thanks @yuxiqian